spec/src/main/asciidoc/configsources.asciidoc

```diff
@@ -35,8 +35,6 @@ It can also be used to implement a drop-in configuration approach.
 Simply create a jar containing a `ConfigSource` with a higher ordinal and override configuration values in it.
 If the jar is present on the classpath then it will override configuration values from <<ConfigSource,ConfigSources>> with lower `ordinal` values.
 
-The ordinal for property file based <<ConfigSource,ConfigSources>> can be configured using the key `config_ordinal` inside the property file.
```

Comment:
> A change like what is being described means that we would need to ask the config source for the value of config_ordinal each time, which is potentially expensive

I really don't think that it implies that the value of config_ordinal is checked each time. It should be checked only once when a config source is created, am I wrong? And checking a single value is negligible to checking all the other values that an application usually requires.

Making support for `config_ordinal` in custom sources optional completely defeats its purpose. If a config source ordinal isn't configurable, it's not easy to configure its order if it clashes with other config sources unless all ordinals in other config sources are modified. 

Therefore I'm against relying solely on the output of the getOrdinal() method, which would optionally return value of the `config_ordinal` property.